src/components/Bale/BaleList.js

```diff
@@ -38,28 +43,53 @@ class BaleList extends Component {
     navBarHidden: true,
   };
 
+  constructor(props) {
+    super(props);
+    this.materials = recyclabeleMaterials;
+  }
+
   render() {
     return (
-      <FlatList
-        data={balesList}
-        renderItem={({ item }) => {
-          if (isPhone) {
-            return <PhoneBale id={item.id} type={item.type} weight={item.weight} />;
-          }
-          return <TabletBale id={item.id} type={item.type} weight={item.weight} />;
-        }}
-      />
+      <View>
+        <CreateBaleModal />
+        <EditBaleModal />
+        <FlatList
+          data={balesList}
+          renderItem={({ item }) => {
+            if (isPhone) {
+              return (
+                <PhoneBale
+                  id={item.id}
+                  onPressAction={this.props.openEditBaleModal}
+                />
```

Comment:
You need to include type and weight in the PhoneBale props too, as the component needs them to show their info when clicked.